app/services/customers/update_service.rb

```diff
@@ -74,6 +74,13 @@ def update(**args)
       ActiveRecord::Base.transaction do
         customer.save!
 
+        if customer.organization.eu_tax_management
+          eu_tax_code = Customers::EuAutoTaxesService.call(customer:)
+
+          args[:tax_codes] ||= []
```

Comment:
I think it will remove taxes for an existing customer if `tax_codes` is not passed but `eu_tax_management` is activated 😬

Reply:
yes, it's the actual behavior we want, we may change this later, the doc will focus on this cc @sarkissianraffi